src_c/_freetype.c

```diff
@@ -2205,10 +2205,9 @@ MODINIT_DEFINE(_freetype)
     FREETYPE_MOD_STATE(module)->cache_size = 0;
     FREETYPE_MOD_STATE(module)->resolution = PGFT_DEFAULT_RESOLUTION;
 
-    Py_INCREF((PyObject *)&pgFont_Type);
-    if (PyModule_AddObject(module, FONT_TYPE_NAME, (PyObject *)&pgFont_Type) ==
-        -1) {
-        Py_DECREF((PyObject *)&pgFont_Type);
+    Py_INCREF(&pgFont_Type);
```

Comment:
`(PyObject *)`?

Reply:
Yes this is intentional. Because the incref/decref API are macros, they cast the argument to `(PyObject *)` already, so doing it before the macro call is redundant